### PR TITLE
Format LVM volumes asynchronously

### PR DESCRIPTION
**Pull request by Zempashi**
Fixes: #173

### CODE REVIEW

File: ChangeLog.rst [hunk 1]
```diff
@@ -13,6 +13,10 @@ upgrading a MetalK8s 0.1 cluster to MetalK8s 0.2 (:ghissue:`147`)
 storage related actions. Please see :ref:`upgrade_from_MetalK8s_before_0.2.0`
 (:ghissue:`153`)
 
+Bugs fixed
```
Comment (NicolasT) [inline ChangeLog.rst]: Not really a bug...

Comment (Zempashi) [reply ChangeLog.rst]: I wondered a long time about this... So it's a feature added with a `ghissue` ?

Comment (NicolasT) [reply ChangeLog.rst]: There are only 'Issues' on Github (and 'Notes' in a 'Project'), so yeah. It's a bit of a misnomer, but we've been using 'Issues' to track features for quite a while already...

**Review by ballot-scality — COMMENTED** [review]
The value of async / poll and retries are not very clear to me.

Also I would very like to have some value in the defaults/main.yml of this role to have a way to modify these values.

**Comment by NicolasT** [comment]
Can this be retargetted to `development/1.0`?

File: ChangeLog.rst [hunk 2]
```diff
@@ -38,6 +38,10 @@ upgrading a MetalK8s 0.1 cluster to MetalK8s 0.2 (:ghissue:`147`)
 storage related actions. Please see :ref:`upgrade_from_MetalK8s_before_0.2.0`
 (:ghissue:`153`)
 
+Features added
```
Comment (NicolasT) [inline ChangeLog.rst]: This should go in the 1.0.0 section

File: ChangeLog.rst [hunk 3]
```diff
@@ -38,6 +38,10 @@ upgrading a MetalK8s 0.1 cluster to MetalK8s 0.2 (:ghissue:`147`)
...
+--------------
+:ghissue:`173` - Format pv in asynchronous mode (:ghpull:`181`)
```
Comment (NicolasT) [inline ChangeLog.rst]:
```
:ghpull:`181` - format `PersistentVolumes` asynchronously (:ghissue:`173`)
```

**Review by NicolasT — CHANGES_REQUESTED** [review]
Needs some ChangeLog fixes.

**Review by NicolasT — DISMISSED** [review]

**Review by NicolasT — APPROVED** [review]

**Pull request merged**